### ISSUE TITLE
Add section about minimum touch target size to buttons and links

### ISSUE THREAD

**Issue by EmmaDawsonDev** [AccessibleForAll/AccessibleWebDev#386]
Since [touch target minimum](https://www.w3.org/TR/WCAG22/#target-size-minimum) was added to WCAG 2.2 a short section should be added about it to the buttons page and the links page

**Comment by CBID2** [comment]
I'll do it @EmmaDawsonDev

**Comment by ctoffanin** [comment]
Hi, yes, go ahead!
